### ISSUE TITLE
Issue with update_ip4log in pfqueue

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#3379]
Getting these errors in the packetfence.log:
```
Jul 11 06:19:25 PFenceSrv18C pfqueue: pfqueue(26057) FATAL: [mac:00:11:22:33:44:55] unable to connect to database: Can't connect to MySQL server on '127.0.0.1' (4 "Interrupted system call") at /usr/local/pf/lib/pf/dal.pm line 70.
 (pf::db::db_connect)
Jul 11 06:19:25 PFenceSrv18C pfqueue: pfqueue(26057) ERROR: [mac:00:11:22:33:44:55] Error handling update_ip4log : unable to connect to database: Can't connect to MySQL server on '127.0.0.1' (4 "Interrupted system call") at /usr/local/pf/lib/pf/dal.pm line 70.
 (pf::api::can_fork::notify)
```

**Comment by julsemaan** [comment]
This is on 8.1

**Comment by julsemaan** [comment]
This was fixed by 7f4c56782676a94aec5211c515497cf695d0668d which is in the maintenance.
